docs/docs/tutorials/cli.md

```diff
@@ -0,0 +1,34 @@
+# CLI
+
+Stoat provides a CLI to help configure Stoat and interact with the comment locally.
+
+## Installation
+
+Requirements:
+- Node/NPM
+- Mac/Linux
+
+To install, run:
+```
+npm i -g stoat
+```
+
+## Initialize project
+
+To initialize a Stoat project within a Git repo:
+```
+stoat init
+```
+
+This will create the Stoat config file and prompt you about adding actions to your Github workflow files.
+We recommend following the prompts and adding the action to any workflow that will push data you want to collect in your comment.
+
+## Local mode
+To preview your comment locally:
+```
+stoat local
+```
+
+This requires that Stoat is already configured for the Git repo. Since it doesn't connect with any build data (it does not send any build data to the Stoat server and doesn't pull in any metric/build data from the server),
```

Comment:
> it does not send any build data to the Stoat server and doesn't pull in any metric/build data from the server

This sentence can be moved upfront and maybe even highlighted, so that users feel safe to use it.